### PR TITLE
Add special event (2021)

### PR DESCRIPTION
**Pull request by francisyzy**
https://appleinsider.com/articles/21/04/13/siri-reveals-apples-next-event-will-be-on-april-20
https://www.macrumors.com/2021/04/13/siri-apple-event-tuesday-april-20/
https://9to5mac.com/2021/04/13/siri-says-the-next-apple-event-is-on-april-20-new-ipad-pro-expected/

### CODE REVIEW

**Comment by stefandesu** [comment]
Now it is also officially announced: https://www.apple.com/apple-events/

**Review by m4xic — COMMENTED** [review]
I would update the name to "Spring Loaded" [since that's the title given by Apple](https://www.macrumors.com/2021/04/13/apple-event-april-23-announcement/)

File: js/addNewEvent.js [hunk 1]
```diff
@@ -21,4 +21,4 @@ minute = 00;
 * UPDATE UPCOMING EVENT NAME
 * --------------------------
 */
-const eventName = 'WWDC 2021';
+const eventName = 'Special Event';
```
Comment (hartator) [inline js/addNewEvent.js:24]:
```suggestion
const eventName = 'Spring Event';
```

File: js/addNewEvent.js [hunk 2]
```diff
@@ -21,4 +21,4 @@ minute = 00;
 * UPDATE UPCOMING EVENT NAME
 * --------------------------
 */
-const eventName = 'WWDC 2021';
+const eventName = 'Spring Event';
```
Comment (hartator) [inline js/addNewEvent.js:24]:
```suggestion
const eventName = 'Spring Loaded';
```

**Comment by hartator** [comment]
Thanks @francisyzy, @stefandesu, @m4xic 👍

Looking forward to it! Hopefully it won't be just about iPad but also more new M1-ish computers. :)

**Review by hartator — APPROVED** [review]

**Comment by stefandesu** [comment]
There seems to be a mistake with the time. http://wheniskeynote.com shows me that the event will be at 20:00 local time (CEST), but the calendar event via https://www.apple.com/apple-events/ is at 19:00. The latter seems to be correct, see [here](https://www.timeanddate.com/worldclock/fixedtime.html?msg=Apple+Event&iso=20210420T10&p1=283).

**Pull request merged**